workflow-templates/integration.yaml

```
@@ -1,17 +1,7 @@
-name: Charmed Kubernetes CI
+name: Charmed Kubernetes Integration Tests
 on: [pull_request]
 
 jobs:
-  call-inclusive-naming-check:
-    name: Inclusive Naming
-    uses: canonical-web-and-design/Inclusive-naming/.github/workflows/woke.yaml@main
-    with:
-      fail-on-error: "true"
-
-  lint-unit:
-    name: Lint Unit
-    uses: charmed-kubernetes/workflows/.github/workflows/lint-unit.yaml@main
-
   integration-test:
```

Should we call it 

```suggestion
  vsphere-integration-test:
```

That way maybe we add different types of integration-tests in here